components/Food/FoodCard.js

```diff
@@ -1,68 +1,66 @@
 import styled from "styled-components";
+import Image from "next/image";
+import FoodImage from "../../assets/images/artichoke.jpg";
+import ArrowIcon from "../../assets/images/arrow_button.svg";
+import Link from "next/link";
 
-export default function FoodCard({ name, img }) {
+export default function FoodCard({ name }) {
   return (
     <StyledItem>
-      <StyledSvg
-        width="11"
-        height="19"
-        viewBox="0 0 11 19"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
-      >
-        <path
-          d="M0.813569 16.4915C0.36211 16.9504 0.362109 17.6866 0.813568 18.1455C1.2756 18.6151 2.03281 18.6151 2.49484 18.1455L9.62017 10.9026C10.3859 10.1243 10.3859 8.87572 9.62017 8.09741L2.49484 0.854508C2.03281 0.384855 1.2756 0.384855 0.813568 0.854508C0.362109 1.31342 0.362109 2.04958 0.813568 2.50849L6.31176 8.0974C7.07744 8.87572 7.07744 10.1243 6.31176 10.9026L0.813569 16.4915Z"
-          fill="#71554F"
-          fill-opacity="0.8"
+      <Link href="/food">
+        <StyledArrowIcon
+          src={ArrowIcon}
+          alt="Arrow Icon"
+          width={11}
+          height={19}
+          priority
         />
-      </StyledSvg>
+      </Link>
+
       <StyledName>{name}</StyledName>
       <StyledMeta>
-        <StyledImg src={img} />
+        <StyledFoodImage
+          src={FoodImage}
+          alt="Food Image"
+          width={60}
+          height={60}
+          priority
```

Comment:
finde mal noch raus, was das bedeutet ;)

Reply:
When true, the image will be considered high priority and [preload](https://web.dev/preload-responsive-images/). Lazy loading is automatically disabled for images using priority.